source/adminguide/tuning.rst

```diff
@@ -109,6 +109,10 @@ db.simulator.connectionPoolLib
 To use DBCP 2, the value for the configuration must be set to 'dbcp'. An
 empty value or 'hikaricp' will allow using HikariCP.
 
+For large-scale environments, HikariCP should perform better. For environments
```

Comment:
Should we also mention, RAM/requirements when running mgmt server with either of the two?

Reply:
@rohityadavcloud while both work fine 2GB JVM heap size (the current default we use) but with HikariCP I've seen failures when the system memory is constrained such as a 4GB system running a management server, usage server and MySQL server. Same failure has been there with DBCP2 as well but very rare. At lower end I've found HikariCP resulting in slightly higher memory usage for management server and MySQL server, a couple of hundreds of MB.
Based on this I'm not sure how I phrase minimum requirements

Reply:
@rohityadavcloud yes 5GB is sweet spot for HikariCP when running all 3 - management server, usage server and MySQL server on same machine but when running DB server separately even 3-4 GB may be enough.
Does this sound okay then?

```
Minimum requirements when running management server, usage server and MySQL server on the same system:
- HikariCP: 2 CPU, 5GB RAM
- DBCP2: 2 CPU, 4GB RAM
```